dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -78,53 +78,54 @@ CAPTION "Apparence avancée"
-    LTEXT "&2ème Couleur", IDC_ADVAPPEARANCE_COLOR2_T, 216, 160, 40, 9
-    CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR2_B, 216, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    LTEXT "Police", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9
+    LTEXT "Couleur 2 :", IDC_ADVAPPEARANCE_COLOR2_T, 215, 160, 34, 9
+    CHECKBOX "-", IDC_ADVAPPEARANCE_COLOR2_B, 218, 170, 28, 13, BS_PUSHLIKE | BS_BITMAP
+    LTEXT "Police :", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9
     COMBOBOX IDC_ADVAPPEARANCE_FONT_C, 5, 200, 130, 90, CBS_DROPDOWN |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Taille", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9
+    LTEXT "Taille :", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9
     COMBOBOX IDC_ADVAPPEARANCE_FONTSIZE_E, 142, 200, 38, 200, CBS_DROPDOWN | WS_VSCROLL
-    LTEXT "Couleur", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9
+    LTEXT "Couleur :", IDC_ADVAPPEARANCE_FONTCOLOR_T, 184, 190, 28, 9
     CHECKBOX "", IDC_ADVAPPEARANCE_FONTCOLOR_B, 184, 200, 28, 13, BS_PUSHLIKE | BS_BITMAP
-    AUTOCHECKBOX "G", IDC_ADVAPPEARANCE_FONTBOLD, 216, 200, 14, 13, BS_PUSHLIKE
-    AUTOCHECKBOX "I ", IDC_ADVAPPEARANCE_FONTITALIC, 230, 200, 14, 13, BS_PUSHLIKE
+    AUTOCHECKBOX "G", IDC_ADVAPPEARANCE_FONTBOLD, 218, 200, 14, 13, BS_PUSHLIKE
+    AUTOCHECKBOX "I ", IDC_ADVAPPEARANCE_FONTITALIC, 232, 200, 14, 13, BS_PUSHLIKE
     PUSHBUTTON "Annuler", IDCANCEL, 191, 220, 50, 14
     DEFPUSHBUTTON "OK", IDOK, 137, 220, 50, 14
 END
 
-IDD_EFFAPPEARANCE DIALOGEX 0, 0, 285, 185
+IDD_EFFAPPEARANCE DIALOGEX 0, 0, 370, 185
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
 CAPTION "Effets"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL "&Utilisez l'effet de transition suivant pour les menus et les bulles d'aides",
+    CONTROL "&Utilisez l'effet de transition suivant pour les menus et les info-bulles :",
             IDC_EFFAPPEARANCE_ANIMATION, "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 5, 267, 19
     COMBOBOX IDC_EFFAPPEARANCE_ANIMATIONTYPE, 20, 25, 95, 19, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | WS_CHILD | WS_VSCROLL | WS_TABSTOP
-    CONTROL "U&tilisez la méthode suivante pour aplanir les bords des polices d'écran", IDC_EFFAPPEARANCE_SMOOTHING,
+    CONTROL "U&tilisez la méthode suivante pour lisser les bords des polices d'écran :", IDC_EFFAPPEARANCE_SMOOTHING,
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 42, 267, 19
     COMBOBOX IDC_EFFAPPEARANCE_SMOOTHINGTYPE, 20, 62, 95, 19, CBS_DROPDOWNLIST |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
     CONTROL "Uti&liser de grandes icones", IDC_EFFAPPEARANCE_LARGEICONS, "button",
             BS_AUTOCHECKBOX | WS_TABSTOP | WS_DISABLED, 10, 80, 267, 19
-    CONTROL "&Montrer ombrage sous les menus", IDC_EFFAPPEARANCE_SETDROPSHADOW,
+    CONTROL "Afficher une ombre sous les menus", IDC_EFFAPPEARANCE_SETDROPSHADOW,
```

Comment:
Ampersand `&` is lost. Is that intended?

Reply:
fixed